### PR TITLE
 DR-1326 Follow up - fix nightly perf test run - looking for incorrect jar file name

### PR DESCRIPTION
**Pull request by snf2ye**
GitHub Action associated PR: https://github.com/broadinstitute/datarepo-actions/pull/33

### CODE REVIEW

File: .github/workflows/test-runner-on-perf.yml [hunk 1]
```diff
@@ -68,7 +69,7 @@ jobs:
           echo "Building Data Repo client library"
           ../gradlew clean assemble
           cd ${GITHUB_WORKSPACE}/${workingDir}/datarepo-clienttests
-          export ORG_GRADLE_PROJECT_datarepoclientjar=$(find .. -type f -name "datarepo-client-*.jar")
+          export ORG_GRADLE_PROJECT_datarepoclientjar=$(find .. -type f -name "datarepo-client*.jar")
```
Comment (marikomedlock) [inline .github/workflows/test-runner-on-perf.yml:71]: This is the only change that will actually be merged right?

**Review by fboulnois — APPROVED** [review]
lgtm

File: .github/workflows/test-runner-on-perf.yml (hunk 1 above)
Comment (snf2ye) [reply .github/workflows/test-runner-on-perf.yml:71]: Correct - This and the GitHub action version bump.

Comment (marikomedlock) [reply .github/workflows/test-runner-on-perf.yml:71]: Got it, thanks.

**Review by marikomedlock — APPROVED** [review]
LGTM

**Pull request merged**